contracts/debt-contracts/TermsContract.sol

```diff
@@ -12,29 +12,40 @@ contract TermsContract is Initializable, ITermsContract, ControllerRole {
     using SafeMath for uint256;
 
     using TermsContractLib for TermsContractLib.LoanParams;
-    using TermsContractLib for TermsContractLib.ScheduledPayment;
     using TermsContractLib for TermsContractLib.LoanStatus;
 
     TermsContractLib.LoanParams public loanParams;
-    TermsContractLib.ScheduledPayment[] public paymentTable;
 
     address private _borrower;
-    // TODO(Dan): To implement
-    // modifier onlyAtStatus(LoanStatus status) {}
 
-    // modifier onlyBeforeStatus(LoanStatus status) {}
+    event RepaymentCycleStarted(uint256 loanStartTimestamp, uint256 principalDisbursed);
```

Comment:
do we need this specific event, since we already have the `LoanSet`, which provides the active status?

Reply:
The start time is a nice piece of info to have in the event IMO.